testdata/simple-go/test_demo.go

```
@@ -0,0 +1,310 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+// Additional constants for testing constant searches
+const (
+	TestConstant    = "test_value"
+	MAX_BUFFER_SIZE = 1024
```

golangci-lint (1.64.8)

ST1003: should not use ALL_CAPS in Go names; use CamelCase instead

(stylecheck)